docs/reference/configuration/mesh.md

```diff
@@ -276,3 +278,69 @@ Name: mbed-mesh-api.6lowpan-nd-security-mode
     Macro name: MBED_CONF_MBED_MESH_API_6LOWPAN_ND_SECURITY_MODE
     Value: NONE (set by library:mbed-mesh-api)
 ```
+
+#### Wi-SUN related configuration parameters
```

Comment:
Configuration should be moved to their own page.
But I don't find the "Configuration" page anymore from os.mbed.com/docs.. it has been very well hidden.

Reply:
But this is the place, were the thread and 6lowpan configurations are

Reply:
Maybe it deserves a separate PR.